auth0/src/main/java/com/auth0/android/provider/WebAuthProvider.java

```diff
@@ -241,9 +258,9 @@ public Builder withAudience(@NonNull String audience) {
         }
 
         /**
-         * Specify a custom Scheme to use on the Callback Uri. Default scheme is 'https'.
+         * Specify a custom Scheme to use on the Redirect Uri. Default scheme is 'https'.
          *
-         * @param scheme to use in the Callback Uri.
+         * @param scheme to use in the Redirect Uri.
```

Comment:
Same as above, let's use URL like the dashboard does.

Reply:
This relates to the name of the param sent on authentication. See https://auth0.com/docs/api/authentication#social. The dashboard uses "Callback URL" (see whitelist box). Will keep as is.